js/languagePickerDrawerView.js

```diff
@@ -29,7 +29,8 @@ define([
       var data = this.model.getLanguageDetails(newLanguage);
 
       var promptObject = {
-        _classes: 'is-ltr',
+        _attributes: { lang: newLanguage },
+        _classes: 'is-ltr is-lang-' + newLanguage,
```

Comment:
```suggestion
        _classes: `is-lang-${newLanguage} ${data._direction === 'rtl' ? 'is-rtl' : 'is-ltr'}`
```
then you can delete lines 49-51